iOS Example Frame/iOS Example Frame/CustomStyle1.swift

```diff
@@ -133,26 +130,26 @@ struct AddBillingDetailsButtonStyleCustom1: ElementButtonStyle {
 
 // MARK: - Expiry Date
 
-public struct ExpiryDateFormStyleCustom1: CellTextFieldStyle {
-  public var isMandatory = true
-  public var backgroundColor: UIColor = .clear
-  public var title: ElementStyle? = TitleLabelStyleCustom1(text: "Expiry date")
-  public var hint: ElementStyle? = HintInputLabelStyleCustom1(text: "Format is MM/YY")
-  public var mandatory: ElementStyle?
-  public var textfield: ElementTextFieldStyle = TextFieldCustom1(text: "test", placeHolder: "MM/YY")
-  public var error: ElementErrorViewStyle? = ErrorInputLabelStyleCustom1(text: "please fill expiry date")
+struct ExpiryDateFormStyleCustom1: CellTextFieldStyle {
+  var isMandatory = true
```

Comment:
Unrelated to this PR : But for Card Number & Expiry this value should always be mandatory by default ( not allow merchants to hide this ). My understanding is that CVV can be empty / optional for few card schemes.

Reply:
Yes that's correct. Card number and expiry date are mandatory, but we should allow Security Code to be an optional field, as merchant's don't actually have to include this.

Reply:
fyi, there is no simple solution for this, as these objects appear more to be designed for internal consumption, not for external. I would add that you have additional similar issues coming from `isHidden` & `isEnabled` which don't really make sense to be inputed by consumer?

Reply:
Please suggest code

Reply:
Good Question @alex-ioja-yang-cko : I think we should review & discuss the purpose of isMandatory, isHidden & isEnabled and then create a separate story to fix if needed. My understanding is as below. 

- `isMandatory` is allowing the merchant to show a asterisk sign on field, Does it have any more logic associated with it ? 
- `isHidden` is allowing a merchant to show and hide a field. In this case we don't need this for expiry date and card number fields as merchant needs these fields for create tokens. 
- `isEnabled` allows merchants to edit a particular text field. I am assuming we dont want our merchants to control when the text fields are enabled or disabled for editing as we are using our validation logic.